### PR TITLE
Add message to log page to indicate there are no transactions

### PR DESCRIPTION
**Pull request by michaelcohen716**
# Description

This PR addresses #1689. When there are no transactions in the log, a message is displayed explaining why the creator doesn't see any records. The message recommends that the creator check back once they've created a lock (I added an add'l sub-message beyond the Figma comp to conform with the style of the `CreatorLocks` dashboard. Let me know if you'd like tweaks to the language.)

Screen Shot 2019-04-07 at 4 51 11 PM

Additionally, this PR adds a loading state to the `CreatorLog` component to prevent the No Transactions message from displaying while the app is querying for the creator's past transactions.

Screen Shot 2019-04-07 at 4 51 48 PM

# Issues

Refs #1689 

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [x] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [x] My code follows the style guidelines of this project, including naming conventions
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
Thanks a lot for your contribution @michaelcohen716 !

File: unlock-app/src/components/creator/CreatorLocks.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import Media, { NoPhone, Phone } from '../../theme/media'
 import { createLock } from '../../actions/lock'
 import { hideForm } from '../../actions/lockFormVisibility'
 import { DefaultError } from './FatalError'
-import Svg from '../interface/svg'
+import Loading from '../interface/Loading'
```
Comment (julien51) [inline unlock-app/src/components/creator/CreatorLocks.js]: +1 for re-factoring and DRY-ing up code ;)

File: unlock-app/src/components/creator/CreatorLog.tsx [hunk 2]
```diff
@@ -1,34 +1,64 @@
...
-            <BlockNumber>{tx.blockNumber}</BlockNumber>
-            <Address href={explorerLinks[tx.hash]} target="_blank">
-              {tx.lock}
-            </Address>
-            <Type type={tx.type}>{tx.type}</Type>
-          </React.Fragment>
-        )
-      })}
-    </Grid>
+    <>
```
Comment (julien51) [inline unlock-app/src/components/creator/CreatorLog.tsx]: Nit: do you mind using `<Fragment>` here? We found that it is slightly less confusing.

**Comment by michaelcohen716** [comment]
pushed some updates! should be ready to go

**Comment by cellog** [comment]
Thanks Michael!

**Pull request merged**